pkg/apis/addtoscheme_operator_v1alpha1.go

```diff
@@ -1,7 +1,7 @@
 package apis
 
 import (
-	"github.com/pmorie/kubefed-operator/pkg/apis/operator/v1alpha1"
+	"github.com/openshift/kubefed-operator/pkg/apis/operator/v1alpha1"
```

Comment:
should we rename these to v1beta1 to be in lock step with the kubefed api versioning?

Reply:
@pmorie any thoughts here?

Reply:
No, the operator's API should still be alpha at this point.